saleor/core/search_tasks.py

```diff
@@ -63,9 +63,12 @@ def set_order_search_document_values(total_count, updated_count):
 
 @app.task
 def set_product_search_document_values(total_count, updated_count):
+    # set lower batch size as it was crashing for products with
+    # lots of attributes because out of memory issues
+    batch_size = 500
```

Comment:
Question: Don't we want to reduce te global batch_size defined in this file? Looks like the logic for `Order` could also fetch a lot of data from related objects.

Reply:
The problem was only for `products` so I reduced it only there. 
Maybe we should consider reducing batch size for orders too, but I do not think it should be reduced as much as for products.